src/components/DocumentationTopic/Hierarchy.vue

```diff
@@ -14,6 +14,18 @@
     :class="{ 'has-badge': hasBadge }"
     class="hierarchy"
   >
+    <slot
+      name="title"
+      v-bind="{ rootLink }"
+    >
+      <HierarchyItem
+        v-if="rootLink"
+        :url="rootLink"
+      >
+        {{ $t('documentation.title') }}
```

Comment:
I understand this may be a design choice, but since we have the standalone "Documentation" Icon now, do we still need the "Documentation" text in the breadcrumb here? I personally think it's a little confusing, since it's not clickable.
<img width="219" alt="image" src="https://github.com/apple/swift-docc-render/assets/87735557/835c5b83-41c1-4ef5-bd41-ff2e51f8c2d3">